Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -179,19 +184,111 @@ final class ChangeEmailViewModelTests: TestCase {
   }
 
   func testResendVerificationStackViewIsHidden_IfEmailIsVerified() {
-    self.resendVerificationStackViewIsHidden.assertDidNotEmitValue()
+    withEnvironment {
```

Comment:
No need for `withEnvironment` unless we're modifying the environment in some way.